pkg/broker/broker.go

```diff
@@ -1004,7 +1004,7 @@ func (a AnsibleBroker) LastOperation(instanceUUID uuid.UUID, req *LastOperationR
 	*/
 	a.log.Debugf("service_id: %s", req.ServiceID)
 	a.log.Debugf("plan_id: %s", req.PlanID)
-	a.log.Debugf("operation:  %s", req.Operation) // this is provided with the provision. task id from the work_engine
```

Comment:
Why is this an erroneous comment? The operation is expected to be the job token (task_id) from the work_engine that happened during the provision. At a minimum I'd remove provision and make it clear that we expect operation to be the job token id.

Reply:
I just remarked it wasn't entirely accurate since all async jobs will set that to be the job token, not just provision.